---
permalink: /
title: "é—µæ–‡æ–‡ä¸ªäººä¸»é¡µ"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

<!--
ä»¥ä¸‹å†…å®¹æ˜¯è¢«æ³¨é‡Šã€‚
1ï¼‰æ”¹å˜å­—ä½“çš„æ–¹å¼
<font face="æ¥·ä½“">
ä¸ªäººç®€ä»‹
</font>
2ï¼‰
<font face="æ¥·ä½“" color="blue" size=5>è¿™æ˜¯è“è‰²5å·æ¥·ä½“</font>
<font color="blue">  è¿™æ˜¯è“è‰²5å·æ¥·ä½“ </font>
&emsp;&emsp;
-->

## ä¸€ã€ä¸ªäººç®€ä»‹
- é—µæ–‡æ–‡ï¼Œäº‘å—å¤§å­¦ä¿¡æ¯å­¦é™¢ï¼Œè®¡ç®—æœºç§‘å­¦ä¸å·¥ç¨‹ç³»å‰¯æ•™æˆï¼Œç¡•å£«ç”Ÿå¯¼å¸ˆï¼Œäº‘å—å¤§å­¦â€œä¸œé™†äººæ‰è®¡åˆ’-é’å¹´å­¦è€…â€ã€‚2017å¹´æ¯•ä¸šäºæ­¦æ±‰å¤§å­¦è®¡ç®—æœºå­¦é™¢ï¼Œè·å·¥å­¦åšå£«å­¦ä½ï¼Œ2019~2021åœ¨é¦™æ¸¯ä¸­æ–‡å¤§å­¦ï¼ˆæ·±åœ³ï¼‰ä»äº‹åšå£«åç ”ç©¶å·¥ä½œï¼Œ2021å¹´12æœˆå…¥é€‰äº‘å—å¤§å­¦â€œä¸œé™†é’å¹´å­¦è€…â€å¼•è¿›äººæ‰ï¼Œ2022å¹´å…¥é€‰äº‘å—çœâ€œå…´æ»‡è‹±æ‰æ”¯æŒè®¡åˆ’â€é’å¹´äººæ‰ã€‚
<!-- ç›®å‰æ‹…ä»»ä¸­å›½è®¡ç®—æœºå­¦ä¼šç”Ÿç‰©ä¿¡æ¯å­¦ä¸“å§”ä¼šå§”å‘˜ï¼Œä¸­å›½ç”Ÿç‰©ä¿¡æ¯å­¦å­¦ä¼š (ç­¹)ç½‘ç»œç”Ÿç‰©å­¦ä¸“å§”ä¼šå§”å‘˜ï¼Œä¸­å›½è¿ç­¹å­¦ä¼šè®¡ç®—ç³»ç»Ÿç”Ÿç‰©å­¦åˆ†ä¼šç†äº‹ï¼ŒInterdisciplinary Sciences: Computational Life Sciencesæ‚å¿—ï¼ˆä¸­ç§‘é™¢äºŒåŒºï¼‰é’å¹´ç¼–è¾‘ï¼Œå›½å®¶è‡ªç„¶ç§‘å­¦åŸºé‡‘é’å¹´ã€é¢ä¸Šé¡¹ç›®å‡½è¯„ä¸“å®¶ç­‰ã€‚ é—µè€å¸ˆé•¿æœŸä»äº‹äººå·¥æ™ºèƒ½ç®—æ³•ï¼ˆå›¾ç¥ç»ç½‘ç»œã€æ·±åº¦æ‰©æ•£æ¨¡å‹ã€Transformeræ¨¡å‹ã€å¯¹æ¯”å­¦ä¹ ã€åŸºäºæ©ç æœºåˆ¶çš„è‡ªç›‘ç£å­¦ä¹ ã€å•ç»†èƒ/ç©ºè½¬å¤§æ¨¡å‹ã€çŸ©é˜µå¼ é‡ä¼˜åŒ–ç­‰ï¼‰åŠå…¶åœ¨ç”Ÿç‰©åŒ»å­¦å¤§æ•°æ®ä¸­çš„åº”ç”¨ç ”ç©¶ã€‚<br> -->
- é—µè€å¸ˆé•¿æœŸä»äº‹äººå·¥æ™ºèƒ½ç®—æ³•ï¼ˆå›¾ç¥ç»ç½‘ç»œã€æ·±åº¦æ‰©æ•£æ¨¡å‹ã€Transformeræ¨¡å‹ã€å¯¹æ¯”å­¦ä¹ ã€åŸºäºæ©ç æœºåˆ¶çš„è‡ªç›‘ç£å­¦ä¹ ã€å•ç»†èƒ/ç©ºè½¬å¤§æ¨¡å‹ã€çŸ©é˜µå¼ é‡ä¼˜åŒ–ç­‰ï¼‰åŠå…¶åº”ç”¨ç ”ç©¶ã€‚
- __å­¦æœ¯å·¥ä½œ__ï¼šå›½å®¶è‡ªç„¶ç§‘å­¦åŸºé‡‘é¡¹ç›®è¯„å®¡ä¸“å®¶ï¼Œæ•™è‚²éƒ¨å­¦ä½ä¸­å¿ƒè¯„å®¡ä¸“å®¶ï¼Œè®¡ç®—æœºæœŸåˆŠ (IEEE TPAMIã€IEEE TKDEã€IEEE TCYBã€IEEE TNNLSã€IEEE TMIã€IEEE TSPã€IEEE TIIã€IEEE TMMã€IEEE JHBIã€IEEE TCSVTã€IEEE TETCIã€IEEE IoTã€IEEE TBMEã€IEEE TCSSã€IEEE SMCAã€IEEE BigDataã€JASã€EAAIã€PRã€ESWAã€Neural Networksã€Medical Image Analysisã€Information Sciencesã€KBSã€ä¸­å›½ç§‘å­¦ç­‰)ï¼Œç”Ÿä¿¡æœŸåˆŠ (Nature Methodsã€Nature Computational Scienceã€Nature Communicationsã€Journal of the American Statistical Associationã€NARã€ASã€CBã€PLOS CBã€BIBã€Bioinformaticsã€IEEE TCBBã€INSCã€Journal of Chemical Information and Modelingç­‰) å®¡ç¨¿ä¸“å®¶ã€‚
- __ç ”ç©¶æˆæœ__ï¼šå‘è¡¨å­¦æœ¯è®ºæ–‡60å¤šç¯‡ï¼Œå…¶ä¸­ç¬¬ä¸€ä½œè€…æˆ–é€šè®¯ä½œè€…å‘è¡¨CCF A-Bç±»æœŸåˆŠ/ä¼šè®®24ç¯‡ï¼ŒåŒ…æ‹¬IEEE TKDE (2021, 2023)ï¼ŒIEEE TNNLS (2022)ï¼ŒBioinformatics (2018, 2023)ï¼ŒBriefings in Bioinformatics (2024a, 2024b, 2025a, 2025b, 2025c), PLoS Computational Biology (2021, 2025a, 2025b)ï¼ŒIEEE TCBB (2018)ï¼ŒIEEE JBHI (2024), IEEE TNB (2016), MICCAI (2025) BIBM (2015, 2023a, 2023b, 2024a, 2024b, 2024c, 2024d, 2024e, 2024f, 2024g), ISBRA (2024a, 2024b, 2024c), ICIC (2019, 2024) ç­‰é‡è¦æœŸåˆŠå’Œä¼šè®®ã€‚ä¸»æŒå›½å®¶è‡ªç„¶ç§‘å­¦åŸºé‡‘é¡¹ç›®2é¡¹ï¼Œä¸­å›½åšå£«åé¢ä¸Šé¡¹ç›®1é¡¹ï¼ŒçœåŸºç¡€ç ”ç©¶é¡¹ç›®2é¡¹ï¼Œâ€œå…´æ»‡è‹±æ‰æ”¯æŒè®¡åˆ’â€é’å¹´äººæ‰é¡¹ç›®1é¡¹ï¼Œäº‘å—çœæ™ºèƒ½ç³»ç»Ÿä¸è®¡ç®—é‡ç‚¹å®éªŒå®¤å¼€æ”¾è¯¾é¢˜ï¼ˆé‡ç‚¹é¡¹ç›®ï¼‰1é¡¹ç­‰ã€‚
<!--
ä¸‹é¢çš„å†…å®¹è¢«æ³¨é‡Šäº†ï¼š
è¿‘äº”å¹´ä¸»è¦ä»äº‹ç”Ÿç‰©ä¿¡æ¯è®¡ç®—ç ”ç©¶ï¼Œä¸»è¦è¯¾é¢˜ä¸ºâ€œé¢å‘ç™Œç—‡ç»„å­¦æ•°æ®èåˆçš„æ™ºèƒ½æ–¹æ³•ç ”ç©¶â€ï¼Œæå‡ºäº†å¤šç§ç»“æ„ç¨€ç–çŸ©é˜µå’Œæ·±åº¦å­¦ä¹ æ™ºèƒ½ç®—æ³•ï¼Œåº”ç”¨äºæ•´åˆåˆ†æå¤šç§ç™Œç—‡çš„å¤šå°ºåº¦ç»„å­¦æ•°æ®ï¼ŒæŒ–æ˜ç™Œç—‡å¼‚å¸¸ç”Ÿç‰©åˆ†å­åŠŸèƒ½æ¨¡å—ï¼Œå¯¹äºå‘ç°ç™Œç—‡é¶æ ‡å…·æœ‰é‡è¦æ„ä¹‰ã€‚å›´ç»•è¯¾é¢˜çš„å…³é”®ç§‘å­¦é—®é¢˜ï¼Œå·²å–å¾—è‰¯å¥½çš„ç§‘ç ”æˆæœï¼Œå‘è¡¨30å¤šç¯‡å­¦æœ¯è®ºæ–‡ï¼Œå…¶ä¸­ç¬¬ä¸€ä½œè€…æˆ–é€šè®¯ä½œè€…å‘è¡¨CCF A/Bç±»æœŸåˆŠ/ä¼šè®®25ç¯‡ï¼ŒåŒ…æ‹¬IEEE TKDE (2021, 2023)ï¼ŒIEEE TNNLS 2022ï¼ŒBioinformatics (2018, 2023)ï¼ŒPLoS Computational Biology 2021ï¼ŒIEEE TCBB 2018ï¼ŒIEEE TNB 2016ç­‰é‡è¦æœŸåˆŠï¼Œä¸»æŒå›½å®¶åŸºé‡‘é¡¹ç›®2é¡¹ï¼Œä¸­å›½åšå£«åé¢ä¸Šé¡¹ç›®1é¡¹ï¼ŒçœåŸºç¡€ç ”ç©¶é¡¹ç›®2é¡¹ï¼Œâ€œå…´æ»‡è‹±æ‰æ”¯æŒè®¡åˆ’â€é’å¹´äººæ‰é¡¹ç›®1é¡¹ï¼Œäº‘å—çœæ™ºèƒ½ç³»ç»Ÿä¸è®¡ç®—é‡ç‚¹å®éªŒå®¤å¼€æ”¾è¯¾é¢˜ï¼ˆé‡ç‚¹é¡¹ç›®ï¼‰1é¡¹ç­‰ã€‚
-->

## äºŒã€ç ”ç©¶æ–¹å‘
<!--
&emsp;&emsp;<font color="blue"> é—µæ–‡æ–‡è€å¸ˆé•¿æœŸä»äº‹äººå·¥æ™ºèƒ½ç®—æ³•ï¼ˆåŸºäºTransformerçš„æ·±åº¦æ–¹æ³•ã€å›¾ç¥ç»ç½‘ç»œã€æ·±åº¦æ‰©æ•£æ¨¡å‹ã€å¯¹æ¯”å­¦ä¹ ã€å•ç»†èƒå¤§æ¨¡å‹ã€çŸ©é˜µå¼ é‡ä¼˜åŒ–ç­‰ï¼‰åŠå…¶åœ¨ç”Ÿç‰©åŒ»å­¦å¤§æ•°æ®åˆ†ææ–¹é¢çš„åº”ç”¨ç ”ç©¶ã€‚</font>
ä¸»è¦ç ”ç©¶æ–¹æ³•å¦‚ä¸‹ï¼š
-->

ğŸ¯äººå·¥æ™ºèƒ½ç†è®ºæ–¹æ³•  <br>
  - æ·±åº¦å­¦ä¹ ç†è®ºæ–¹æ³•ï¼ˆå›¾ç¥ç»ç½‘ç»œã€æ·±åº¦æ‰©æ•£æ¨¡å‹ã€Transformerã€å¯¹æ¯”å­¦ä¹ ã€å¤§æ¨¡å‹ï¼‰<br>
  - æœºå™¨å­¦ä¹ ç†è®ºæ–¹æ³•ï¼ˆçŸ©é˜µå¼ é‡æ–¹æ³•ï¼Œéå‡¸ä¼˜åŒ–ã€L20åŠL0ç»“æ„ç¨€ç–ä¼˜åŒ–ã€ç‰¹å¾é€‰æ‹©ï¼‰<br>
  - å¤šæ¨¡æ€ä¹‹èåˆå­¦ä¹ ï¼ˆå›¾ç¥ç»ç½‘ç»œï¼Œæ‰©æ•£æ¨¡å‹ï¼Œæ·±åº¦å¯¹æ¯”å­¦ä¹ ï¼‰ <br>

ğŸ¯åŒ»å­¦äººå·¥æ™ºèƒ½æ–¹æ³• <br>
  - AI + åŒ»å­¦å›¾åƒ <br>
  - AI + å¤šæ¨¡æ€ç”Ÿç‰©åŒ»å­¦æ•°æ® <br>

ğŸ¯  <font color="red"> æ‹›ç”Ÿå¹¿å‘Š  </font>
- æœ¬äººç›®å‰å¯ä»¥æ‹›æ”¶è®¡ç®—æœºç›¸å…³ä¸“ä¸šçš„ä¸“ç¡•ã€å­¦ç¡•ï¼Œç›®å‰ä¸è€ƒè™‘æ‹›ç”Ÿå¤–ç±å­¦ç”Ÿ
- AI+æ•°æ®æŒ–æ˜çš„å›¢é˜Ÿï¼Œä¸é™å®šæ•°æ®ç±»å‹ï¼ŒæŒ–æ˜çš„æ•°æ®å¯¹è±¡å¯åŒ…æ‹¬ï¼šåŒ»å­¦å›¾åƒæ•°æ®ï¼Œç”Ÿç‰©ç»„å­¦æ•°æ®ï¼Œè‡ªç„¶å›¾åƒæ•°æ®ï¼Œç½‘ç»œç»“æ„æ•°æ®ç­‰ï¼Œä¸éœ€è¦ä»»ä½•ç”Ÿç‰©åŒ»å­¦èƒŒæ™¯ï¼Œä¸“æ³¨äºäººå·¥æ™ºèƒ½çš„æ–°ç®—æ³•å¼€å‘ï¼ï¼ï¼
- æ‹›ç”Ÿè¦æ±‚: åšæŒä¸æ‡ˆçš„æ¯…åŠ›ï¼ˆæœ€çœ‹é‡çš„å“è´¨ï¼‰ï¼›è‰¯å¥½çš„ç¼–ç¨‹ã€æ•°å­¦ã€è‹±è¯­èƒ½åŠ›
- <font color="red"> æ„Ÿå…´è¶£çš„åŒå­¦å¯ä»¥ç»™æˆ‘é‚®ä»¶(wenwen.min@qq.com, minwenwen@ynu.edu.cn)ï¼›é‚®ä»¶å†…å®¹åŒ…æ‹¬å¦‚ä¸‹ä¿¡æ¯ï¼šä¸ªäººç®€å†+æœ¬ç§‘æˆç»©å•+è€ƒç ”æˆç»©+å¤è¯•ä¸Šæœºæˆç»©ã€‚ </font>
<!--
(2) æ·±åº¦å­¦ä¹ åœ¨ç©ºé—´è½¬å½•ç»„å­¦åŠå•ç»†èƒç»„å­¦æ•°æ®èåˆä¸­çš„åº”ç”¨ç ”ç©¶
(3) æ·±åº¦å­¦ä¹ åœ¨åŒ»å­¦å›¾åƒå¤„ç†åŠå¤šæ¨¡æ€æ•°æ®èåˆä¸­çš„åº”ç”¨ç ”ç©¶<br>
(4) æ·±åº¦å­¦ä¹ åœ¨ç™Œç—‡ç»„å­¦æ•°æ®åˆ†æä¸­çš„åº”ç”¨ç ”ç©¶<br>
(5) æ·±åº¦å­¦ä¹ åœ¨å¾®ç”Ÿç‰©ç»„å­¦æ•°æ®åˆ†æä¸­çš„åº”ç”¨ç ”ç©¶<br>
-->

## ä¸‰ã€å­¦æœ¯è®ºæ–‡ 

- è°·æ­Œå­¦æœ¯ä¸»é¡µ: [Full List at My Google Scholar](https://scholar.google.com/citations?user=0Uy0GnoAAAAJ&hl=en) 
- ORCIDä¸»é¡µ: [My ORCID](https://orcid.org/0000-0002-2558-2911)
- ResearchGateä¸»é¡µ: [My ResearchGate](https://www.researchgate.net/profile/Wenwen-Min)


## å››ã€ä»£è¡¨ä½œ

- [01] SpaBatch: Deep learning-based cross-slice integration and 3D spatial domain identification in spatial transcriptomics, __<u> Advanced Science 2025 </u>__  (ä¸­ç§‘é™¢ä¸€åŒº, IF=14.3) __(å”¯ä¸€é€šè®¯ä½œè€…, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpaBatch))__
  
- [02] SpaCross deciphers spatial structures and corrects batch effects in multi-slice spatially resolved transcriptomics,
  __<u> Communications Biology </u>__ 2025 (Natureå°å­åˆŠ, ä¸­ç§‘é™¢ä¸€åŒº, IF=5.3)__(å”¯ä¸€é€šè®¯ä½œè€…, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpaCross))__

- [03] __Wenwen Min*__, Taosheng Xu, Xiang Wan, Tsung-Hui Chang. Structured Sparse Non-negative Matrix Factorization with L20-Norm. __<u>IEEE Transactions on Knowledge and Data Engineering</u>__,
  35(8):8584-8595, 2023 (ä¸­ç§‘é™¢ä¸€åŒº, CCF Aç±»æœŸåˆŠ, IF=10.4) [[ä¸‹è½½](https://ieeexplore.ieee.org/document/9893402)] <br>
  
- [04] __Wenwen Min__, Juan Liu\*, Shihua Zhang*. Group-sparse SVD Models via L0 and L1-norm Penalties and Their Applications in Biological Data.
  __<u>IEEE Transactions on Knowledge and Data Engineering</u>__, 33(2):536-550, 2021 (ä¸­ç§‘é™¢ä¸€åŒº, CCF Aç±»æœŸåˆŠ, IF=10.4) [[ä¸‹è½½](https://ieeexplore.ieee.org/document/8782829)]<br>

- [05] __Wenwen Min__, Xiang Wan, Tsung-Hui Chang, Shihua Zhang*. A Novel Sparse Graph-Regularized Singular Value Decomposition Model for Gene Co-Expression Pattern Discovery. __<u>IEEE Transactions on Neural Networks and Learning Systems</u>__, 33(8):3842-3856, 2022 (ä¸­ç§‘é™¢ä¸€åŒºï¼ŒCCF Bç±»æœŸåˆŠ, IF=14.255) [[ä¸‹è½½](https://ieeexplore.ieee.org/document/9350193)]<br>

- [06] Weighted sparse partial least squares for joint sample and feature selection. __<u> IEEE Transactions on Computational Biology and Bioinformatics </u>__ 2025 (ä¸­ç§‘é™¢äºŒåŒºï¼Œ CCF Bç±»æœŸåˆŠ) (ä¸€ä½œ+é€šè®¯ä½œè€…, æ–¹æ³•é¢†åŸŸï¼šæœºå™¨å­¦ä¹ ï¼‰

- [07] SpaMask: Dual masking graph autoencoder with contrastive learning for spatial transcriptomics.  __<u> PLOS Computational Biology 2025 </u>__(é¢†åŸŸé¡¶åˆŠ, ä¸­ç§‘é™¢äºŒåŒºï¼ŒCCF B) (å”¯ä¸€é€šè®¯ä½œè€…, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpaMask))
  
- [08] SpaDAMA: Improving Cell-Type Composition Inference in Spatial Transcriptomics with Domain-Adversarial Masked Autoencoder,
  __<u> PLOS Computational Biology 2025 </u>__ (é¢†åŸŸé¡¶åˆŠ, ä¸­ç§‘é™¢äºŒåŒºï¼ŒCCF B) __(å”¯ä¸€é€šè®¯ä½œè€…, æ–¹æ³•é¢†åŸŸï¼šå¯¹æŠ—ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpaDAMA))__

- [09] __Wenwen Min__, Tsung-Hui Chang, Shihua Zhang*, Xiang Wan*.  TSCCA: A tensor sparse CCA method for detecting microRNA-gene patterns from multiple cancers.  __<u>PLoS Computational Biology</u>__, 
  17(6):e1009044, 2021 (é¢†åŸŸé¡¶åˆŠ, ä¸­ç§‘é™¢äºŒåŒºï¼ŒCCF B, IF=4.779)

- [10] Shuailin Xue, Changmiao Wang, Xiaomao Fan and __Wenwen Min*__. Inferring Super-Resolved Gene Expression by Integrating Histology Images and Spatial Transcriptomics with HISTEX. __<u> MICCAI 2025 </u>__ (CCF B, åŒ»å­¦å›¾åƒè®¡ç®—é¡¶ä¼šï¼ŒMICCAI2025 Top 2%) __(Best Paper, å”¯ä¸€é€šè®¯ä½œè€…, æ–¹æ³•é¢†åŸŸï¼šæ³¨æ„åŠ›æœºåˆ¶ [GitHub](https://github.com/wenwenmin/HISTEX))__

    
<!--
## å››ã€ä¸»è¦è®ºæ–‡(ä¸€ä½œ+é€šè®¯)
#åŒç­‰è´¡çŒ®ä½œè€…ï¼Œ*é€šè®¯ä½œè€…
### ğŸ”¥ 2025

- [45] Geometry-informed Multimodal Fusion Network for Enhancing High-density Spatial Transcriptomics from Histology Images. Engineering Applications of Artificial Intelligence 2025 (Minor Revisionï¼Œä¸­ç§‘é™¢ä¸€åŒº)__(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/HisHRST))__
  
- [44] Weighted sparse partial least squares for joint sample and feature selection. __<u> IEEE Transactions on Computational Biology and Bioinformatics </u>__ 2025 (CCF B, å¯¼å¸ˆç‹¬ç«‹å®Œæˆï¼Œæ— å­¦ç”Ÿå‚ä¸)
  
- [43] SpaBatch: Deep learning-based cross-slice integration and 3D spatial domain identification in spatial transcriptomics, __<u> Advanced Science </u>__ 2025 (IF=14.3, ä¸­ç§‘é™¢ä¸€åŒº) (å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpaBatch))__
  
- [42] SpaCross: Deciphering Spatial Structures and Correcting Batch Effects of Multiâ€‘slice Spatially Resolved Transcriptomics,
  __<u> Communications Biology </u>__ 2025 (Natureå°å­åˆŠ, ä¸­ç§‘é™¢ä¸€åŒº)__(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpaCross))__

- [41] J Zhao and __Wenwen Min*__. SpaICL: Image-Guided Curriculum Strategy-Based Graph Contrastive Learning for Spatial Transcriptomics Clustering.
 __<u> Briefings in Bioinformatics 2025 </u>__ (CCF Bç±»æœŸåˆŠ, ä¸­ç§‘é™¢ä¸€åŒº, IF=9.5) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾å¯¹æ¯”å­¦ä¹  [GitHub](https://github.com/wenwenmin/SpaICL))__
  
- [40] L Huang, X Liu, F Zhuand __Wenwen Min*__. SpaDAMA: Improving Cell-Type Composition Inference in Spatial Transcriptomics with Domain-Adversarial Masked Autoencoder,
  PLOS Computational Biology 2025 (é¢†åŸŸé¡¶åˆŠ, ä¸­ç§‘é™¢äºŒåŒºï¼ŒCCF B) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå¯¹æŠ—ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpaDAMA))__

- [39] Shuailin Xue, Changmiao Wang, Xiaomao Fan and __Wenwen Min*__. Inferring Super-Resolved Gene Expression by Integrating Histology Images and Spatial Transcriptomics with HISTEX. __<u> MICCAI 2025 </u>__ (CCF B, åŒ»å­¦å›¾åƒè®¡ç®—é¡¶ä¼šï¼ŒMICCAI2025 Top 9%) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ³¨æ„åŠ›æœºåˆ¶ [GitHub](https://github.com/wenwenmin/HISTEX))__
  
- [38] Shuailin Xue, F Zhu, J Chen, __Wenwen Min*__. Inferring single-cell resolution spatial gene expression via fusing spot-based spatial transcriptomics, location and histology using GCN.  __<u> Briefings in Bioinformatics 2025 </u>__, DOI: 10.1093/bib/bbae630 (CCF Bç±»æœŸåˆŠ, ä¸­ç§‘é™¢ä¸€åŒº, IF=9.5) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/scstGCN))__
  
- [37] Z Shi, F Zhu, , __Wenwen Min*__. Inferring Multi-slice Spatially Resolved Gene Expression from H&E-stained Histology Images with STMCL.  __<u> Methods </u>__, 2025 (ä¸­ç§‘é™¢ä¸‰åŒº, IF=4.2) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šäººå·¥æ™ºèƒ½ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦å¯¹æ¯”å­¦ä¹  [GitHub](https://github.com/wenwenmin/STMCL))__ 

- [36] X Shi, F Zhu, __Wenwen Min*__. VTrans: A VAE-based Pre-trained Transformer Method for Microbiome Data Analysis. __<u> Journal of Computational Biology 2025 </u>__ (ä¸­ç§‘é™¢ä¸‰åŒº, é¢†åŸŸé¡¶åˆŠ) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šäººå·¥æ™ºèƒ½ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šTransformer [GitHub](https://github.com/wenwenmin/VTrans))__ 

- [35] W Yang, __Wenwen Min*__. Graph regularized sparse nonnegative Tucker decomposition with L0-constraints for unsupervised learning. __<u> Chinese Journal of Electronics 2025 </u>__ (CCF T1, T1ä¸ºæœ€é¡¶çº§æœŸåˆŠ) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºåº”ç”¨æŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæœºå™¨å­¦ä¹ ä¼˜åŒ– [GitHub](https://github.com/wenwenmin/L0GSNTD))__  

- [34] W Yang, __Wenwen Min*__. Sparse nonnegative CP decomposition with graph regularization and L0-constraints for clustering. __<u> Chinese Journal of Electronics 2025 </u>__ (CCF T1, T1ä¸ºæœ€é¡¶çº§æœŸåˆŠ) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºåº”ç”¨æŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæœºå™¨å­¦ä¹ ä¼˜åŒ– [GitHub](https://github.com/wenwenmin/SGNCP))__ 

- [33] J Chen*, __Wenwen Min*__. sTPLS: identifying common and specific correlated patterns under multiple biological conditions. __<u> Briefings in Bioinformatics 2025 </u>__, DOI: 10.1093/bib/bbaf195 (CCF Bç±»æœŸåˆŠ, ä¸­ç§‘é™¢ä¸€åŒº, IF=9.5) __(æ–¹æ³•é¢†åŸŸï¼šå¼ é‡åˆ†è§£ [GitHub](https://github.com/Jinyu2019/sTPLS))__

- [32] __Wenwen Min*__, X Shi, et. al. SpaViT: Self-supervised Prediction of High-Resolution Spatial Transcriptomics with Vision Transformer. __<u> Tsinghua Science and Technology 2025 </u>__ (ä¸­ç§‘é™¢ä¸€åŒº) __(å­¦ç”ŸäºŒä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šTransformer [GitHub](https://github.com/wenwenmin/SpaViT))__

- [31] __Wenwen Min*__, D Fang#, J Chen, S Zhang. SpaMask: Dual masking graph autoencoder with contrastive learning for spatial transcriptomics.  __<u> PLOS Computational Biology 2025 </u>__, DOI: 10.1371/journal.pcbi.1012881 (CCF Bç±»æœŸåˆŠ, é¢†åŸŸé¡¶åˆŠ) __(å­¦ç”Ÿå¹¶ä¸€ä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šè®¡ç®—æœºåº”ç”¨æŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpaMask))__


### ğŸ”¥ 2024

- [30] Xiaoyu Li and __Wenwen Min*__. SpaDiT: Diffusion Transformer for Spatial Gene Expression Prediction using scRNA-seq. __<u> Briefings in Bioinformatics 2024</u>__, DOI: 10.1093/bib/bbae571 (CCF Bç±»æœŸåˆŠ, ä¸­ç§‘é™¢ä¸€åŒº, IF=9.5) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºåº”ç”¨æŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦æ‰©æ•£æ¨¡å‹ [GitHub](https://github.com/wenwenmin/SpaDiT))__
  
- [29] Jinyun Niu and __Wenwen Min*__. Deep Clustering Representation for Spatially Resolved Transcriptomics Data via Multi-view Variational Graph Auto-Encoders with Consensus Clustering. __<u> Computational and Structural Biotechnology Journal 2024</u>__ (ä¸­ç§‘é™¢äºŒåŒº, IF=4.4) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/STMVGAE))__

- [28] J Niu, F Zhu, D Fang, __Wenwen Min*__. SpatialCVGAE: Consensus Clustering Improves Spatial Domain Identification of Spatial Transcriptomics using VGAE. __<u> Interdisciplinary Sciences: Computational Life Sciences 2024  </u>__ (ä¸­ç§‘é™¢äºŒåŒº, IF=4.1)  __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/SpatialCVGAE))__

- [27] __Wenwen Min*__, Zhiceng Shi, Jun Zhang, Jun Wan and Changmiao Wang. Multimodal contrastive learning for spatial gene expression prediction using histology images. __<u> Briefings in Bioinformatics 2024 </u>__, DOI: 10.1093/bib/bbae551 (CCF Bç±»æœŸåˆŠ, ä¸­ç§‘é™¢ä¸€åŒº, IF=9.5) __(å­¦ç”ŸäºŒä½œå¯¼å¸ˆé€šè®¯, å­¦ç”Ÿï¼šäººå·¥æ™ºèƒ½ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦å¯¹æ¯”å­¦ä¹  [GitHub](https://github.com/wenwenmin/mclSTExp))__
  
- [26] X Shi, __Wenwen Min*__. Pretrained-Guided Conditional Diffusion Models for Microbiome Data Analysis. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: äººå·¥æ™ºèƒ½ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦æ‰©æ•£æ¨¡å‹ [GitHub](https://github.com/wenwenmin/mbVDiT))__

- [25] D Fang, __Wenwen Min*__. Multi-Slice Spatial Transcriptomics Data Integration Analysis with STG3Net. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºåº”ç”¨æŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦å¯¹æŠ—å­¦ä¹  [GitHub](https://github.com/wenwenmin/STG3Net))__

- [24] D Fang and __Wenwen Min*__. Masked Graph Autoencoders with Contrastive Augmentation for Spatially Resolved Transcriptomics Data. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºåº”ç”¨æŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦å¯¹æ¯”å­¦ä¹  [GitHub](https://github.com/wenwenmin/STMGAC))__

- [23] __Wenwen Min*__, Z Wang, et. al. scASDC: Attention Enhanced Structural Deep Clustering for Single-cell RNA-seq Data. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%) __(å­¦ç”ŸäºŒä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºæŠ€æœ¯, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/scASDC))__

- [22] Zhiceng Shi and __Wenwen Min*__. High-Resolution Spatial Transcriptomics from Histology Images using HisToSGE with Pathology Image Large Model. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: äººå·¥æ™ºèƒ½ä¸“ä¸šä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šTransformer+å¤§æ¨¡å‹ [GitHub](https://github.com/wenwenmin/HisToSGE))__
 
- [21] Xiaoyu Li, __Wenwen Min*__, Shunfang Wang, Changmiao Wang and Taosheng Xu. stMCDI: Masked Conditional Diffusion Model with Graph Neural Network for Spatial Transcriptomics Data Imputation. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºåº”ç”¨æŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦æ‰©æ•£æ¨¡å‹ [GitHub](https://github.com/wenwenmin/stMCDI))__

- [20] L Huang, X Liu, S Wang, __Wenwen Min*__. Masked adversarial neural network for cell type deconvolution in spatial transcriptomics. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºè½¯ä»¶ä¸ç†è®ºä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦å¯¹æŠ—å­¦ä¹  [GitHub](https://github.com/wenwenmin/MACD))__

- [19] Donghai Fang and __Wenwen Min*__. Contrastive Masked Graph Autoencoders for Spatial Transcriptomics Data Analysis. __<u> ISBRA 2024 </u>__ (CCF Cç±»ä¼šè®®) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºåº”ç”¨æŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦å¯¹æ¯”å­¦ä¹  [GitHub](https://github.com/wenwenmin/STMGAC))__

- [18] Zhiceng Shi and __Wenwen Min*__. Spatial Gene Expression Prediction from Histology Images with STco.__<u> ISBRA 2024 </u>__ (CCF Cç±»ä¼šè®®) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: äººå·¥æ™ºèƒ½ä¸“ä¸šä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šæ·±åº¦å¯¹æ¯”å­¦ä¹  [GitHub](https://github.com/wenwenmin/STco))__

- [17] Shuailin Xue and __Wenwen Min*__. stEnTrans: Transformer-based deep learning for spatial transcriptomics enhancement. __<u> ISBRA 2024 </u>__ (CCF Cç±»ä¼šè®®) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºæŠ€æœ¯ä¸“ä¸š, æ–¹æ³•é¢†åŸŸï¼šå›¾ç¥ç»ç½‘ç»œ [GitHub](https://github.com/wenwenmin/stEnTrans))__
  
- [16] F. Yang, __Wenwen Min*__. Globally Convergent Accelerated Algorithms for Multilinear Sparse Logistic Regression with L0-Constraints. __<u>International Conference on Intelligent Computing (ICIC)}</u>__, 88-99, 2024 (CCF Cç±»ä¼šè®®) __(å­¦ç”Ÿä¸€ä½œå¯¼å¸ˆé€šè®¯ï¼Œå­¦ç”Ÿ: è®¡ç®—æœºåº”ç”¨æŠ€æœ¯, æ–¹æ³•é¢†åŸŸï¼šæœºå™¨å­¦ä¹ ä¼˜åŒ– [GitHub](https://github.com/wenwenmin/stEnTrans))__

- [15] Li Yu#, __Wenwen Min#__, Shunfang Wang\*. Boundary-Aware Gradient Operator Network for Medical Image Segmentation.
  __<u>IEEE Journal of Biomedical and Health Informatics</u>__,
  2024 (ä¸­ç§‘é™¢ä¸€åŒº, CCF Cç±»æœŸåˆŠ, IF=7.7) __(åšå­¦ç”Ÿ, æ–¹æ³•é¢†åŸŸï¼šåŒ»å­¦å›¾åƒåˆ†å‰² [GitHub](https://github.com/LiYu51/BG-Net))__

### ğŸ”¥ 2023

- [14] Xiaoyu Li, Taosheng Xu, Jinyu Chen, Jun Wan and __Wenwen Min*__(å­¦ç”Ÿä¸€ä½œ). 
  Multimodal attention-based variational autoencoder for clinical risk prediction. 
  __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2023 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%)

- [13] Xiaoyu Li, __Wenwen Min*__(å­¦ç”Ÿä¸€ä½œ), Jinyu Chen, Jiaxin Wu and Shunfang Wang.
  TransVCOX: Bridging Transformer Encoder and Pre-trained VAE for Robust Cancer Multi-Omics Survival Analysis.
  __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2023 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%)

- [12] Weihua Zheng#, __Wenwen Min#__, Shunfang Wang\*. TsImpute: An accurate two-step imputation method for single-cell RNA-seq data.
  __<u>Bioinformatics</u>__,
  39(12):btad731, 2023 (CCF Bç±»æœŸåˆŠ, IF=5.8)

- [11] __Wenwen Min*__, Taosheng Xu, Xiang Wan, Tsung-Hui Chang.
  Structured Sparse Non-negative Matrix Factorization with L20-Norm.
  __<u>IEEE Transactions on Knowledge and Data Engineering</u>__,
  35(8):8584-8595, 2023
  (CCF Aç±»æœŸåˆŠ, IF=9.235) [[URL](https://ieeexplore.ieee.org/document/9893402),PDF,Data,Code] <br>

### ğŸ”¥ 2022

- [10] __Wenwen Min__, Xiang Wan, Tsung-Hui Chang, Shihua Zhang*.
  A Novel Sparse Graph-Regularized Singular Value Decomposition Model for Gene Co-Expression Pattern Discovery.
  __<u>IEEE Transactions on Neural Networks and Learning Systems</u>__,
  33(8):3842-3856, 2022
  (ä¸­ç§‘é™¢ä¸€åŒºï¼ŒCCF Bç±»æœŸåˆŠ, IF=14.255) [[URL](https://ieeexplore.ieee.org/document/9350193),PDF,Data,Code]<br>
  
- [09] __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Group-sparse SVD Models via L0 and L1-norm Penalties and Their Applications in Biological Data.
  __<u>IEEE Transactions on Knowledge and Data Engineering</u>__, 
  33(2):536-550, 2021 (CCF Aç±»æœŸåˆŠ, IF=9.235) [[URL](https://ieeexplore.ieee.org/document/8782829),PDF,Data,Code]<br>

- [08] __Wenwen Min__, Tsung-Hui Chang, Shihua Zhang\*, Xiang Wan*. 
  TSCCA: A tensor sparse CCA method for detecting microRNA-gene patterns from multiple cancers. 
  __<u>PLoS Computational Biology</u>__, 
  17(6):e1009044, 2021 
  (CCF Bç±»æœŸåˆŠ, IF=4.779)

- [07] F. Zhu*, J. Li*, J. Liu*, __Wenwen Min*__. Network-based cancer genomic data integration for pattern discovery. __<u>BMC Genomic Data</u>__, 22: Article 54, 2021 (SCIäºŒåŒº)
  
- [06] F. Zhu, J. Liu, __Wenwen Min*__. Gene Functional Module Discovery via Integrating Gene Expression and PPI Network Data. __<u>International Conference on Intelligent Computing (ICIC)}</u>__, 116-126, 2019 (CCF Cç±»ä¼šè®®)

- [05] __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Edge-group sparse PCA for network-guided high dimensional data analysis.
  __<u>Bioinformatics</u>__,
  34(20):3479-3487, 2018
  (CCF Bç±»æœŸåˆŠ, IF=6.931)

- [04] __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Network-regularized sparse logistic regression models for clinical risk prediction and biomarker discovery.
    __<u>IEEE/ACM Transactions on Computational Biology and Bioinformatics</u>__,
  15(3):944-953, 2018
  (CCF Bç±»æœŸåˆŠ, IF=4.5)

- [03] __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Sparse weighted canonical correlation Analysis.
  __<u>Chinese Journal of Electronics (ç”µå­å­¦æŠ¥è‹±æ–‡ç‰ˆ)</u>__,
  27(3):459-466, 2018 ï¼ˆCCF Top Iç±»æœŸåˆŠï¼‰

- [02] __Wenwen Min__, Juan Liu\*, Fei Luo, Shihua Zhang*.
  A two-stage method to identify joint modules from matched microRNA and mRNA expression data.
    __<u>IEEE Transactions on Nanobioscience</u>__,
  15(4):362-370, 2016 

- [01] __Wenwen Min__, Juan Liu*, Fei Luo, Shihua Zhang*.
  A novel two-stage method for identifying microRNA- gene regulatory modules in breast cancer.
    __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2015 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%ï¼‰

-->






