---
permalink: /
title: "é—µæ–‡æ–‡ä¸ªäººä¸»é¡µ"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

<!--
ä»¥ä¸‹å†…å®¹æ˜¯è¢«æ³¨é‡Šã€‚
1ï¼‰æ”¹å˜å­—ä½“çš„æ–¹å¼
<font face="æ¥·ä½“">
ä¸ªäººç®€ä»‹
</font>
2ï¼‰
<font face="æ¥·ä½“" color="blue" size=5>è¿™æ˜¯è“è‰²5å·æ¥·ä½“</font>
<font color="blue">  è¿™æ˜¯è“è‰²5å·æ¥·ä½“ </font>
&emsp;&emsp;
-->

## ä¸€ã€ä¸ªäººç®€ä»‹
- é—µæ–‡æ–‡ï¼Œäº‘å—å¤§å­¦ä¿¡æ¯å­¦é™¢ï¼Œè®¡ç®—æœºç§‘å­¦ä¸å·¥ç¨‹ç³»å‰¯æ•™æˆï¼Œç¡•å£«ç”Ÿå¯¼å¸ˆï¼Œäº‘å—å¤§å­¦â€œä¸œé™†äººæ‰è®¡åˆ’-é’å¹´å­¦è€…â€ã€‚2017å¹´æ¯•ä¸šäºæ­¦æ±‰å¤§å­¦è®¡ç®—æœºå­¦é™¢ï¼Œè·å·¥å­¦åšå£«å­¦ä½ï¼Œ2019~2021åœ¨é¦™æ¸¯ä¸­æ–‡å¤§å­¦ï¼ˆæ·±åœ³ï¼‰ä»äº‹åšå£«åç ”ç©¶å·¥ä½œï¼Œ2021å¹´12æœˆå…¥é€‰äº‘å—å¤§å­¦â€œä¸œé™†é’å¹´å­¦è€…â€å¼•è¿›äººæ‰ï¼Œ2022å¹´å…¥é€‰äº‘å—çœâ€œå…´æ»‡è‹±æ‰æ”¯æŒè®¡åˆ’â€é’å¹´äººæ‰ã€‚ç›®å‰æ‹…ä»»ä¸­å›½è®¡ç®—æœºå­¦ä¼šç”Ÿç‰©ä¿¡æ¯å­¦ä¸“å§”ä¼šå§”å‘˜ï¼Œä¸­å›½ç”Ÿç‰©ä¿¡æ¯å­¦å­¦ä¼š (ç­¹)ç½‘ç»œç”Ÿç‰©å­¦ä¸“å§”ä¼šå§”å‘˜ï¼Œä¸­å›½è¿ç­¹å­¦ä¼šè®¡ç®—ç³»ç»Ÿç”Ÿç‰©å­¦åˆ†ä¼šç†äº‹ï¼ŒInterdisciplinary Sciences: Computational Life Sciencesæ‚å¿—ï¼ˆä¸­ç§‘é™¢äºŒåŒºï¼‰é’å¹´ç¼–è¾‘ï¼Œå›½å®¶è‡ªç„¶ç§‘å­¦åŸºé‡‘é’å¹´ã€é¢ä¸Šé¡¹ç›®å‡½è¯„ä¸“å®¶ç­‰ã€‚<br>
- é—µè€å¸ˆé•¿æœŸä»äº‹äººå·¥æ™ºèƒ½ç®—æ³•ï¼ˆå›¾ç¥ç»ç½‘ç»œã€æ·±åº¦æ‰©æ•£æ¨¡å‹ã€Transformeræ¨¡å‹ã€å¯¹æ¯”å­¦ä¹ ã€åŸºäºæ©ç æœºåˆ¶çš„è‡ªç›‘ç£å­¦ä¹ ã€å•ç»†èƒ/ç©ºè½¬å¤§æ¨¡å‹ã€çŸ©é˜µå¼ é‡ä¼˜åŒ–ç­‰ï¼‰åŠå…¶åœ¨ç”Ÿç‰©åŒ»å­¦å¤§æ•°æ®ä¸­çš„åº”ç”¨ç ”ç©¶ã€‚
- å‘è¡¨å­¦æœ¯è®ºæ–‡è¿‘50ç¯‡ï¼Œå…¶ä¸­ç¬¬ä¸€ä½œè€…æˆ–é€šè®¯ä½œè€…å‘è¡¨CCF A-Bç±»æœŸåˆŠ/ä¼šè®®22ç¯‡ï¼ŒåŒ…æ‹¬IEEE TKDE (2021, 2023)ï¼ŒIEEE TNNLS (2022)ï¼ŒBioinformatics (2018, 2023)ï¼ŒBriefings in Bioinformatics (2024a, 2024b, 2025a, 2025b), PLoS Computational Biology (2021)ï¼ŒIEEE TCBB (2018)ï¼ŒIEEE JBHI (2024), IEEE TNB (2016), BIBM (2015, 2023a, 2023b, 2024a, 2024b, 2024c, 2024d, 2024e, 2024f, 2024g), ISBRA (2024a, 2024b, 2024c), ICIC (2019, 2024) ç­‰é‡è¦æœŸåˆŠå’Œä¼šè®®ã€‚
- ä¸»æŒå›½å®¶åŸºé‡‘é¡¹ç›®2é¡¹ï¼Œä¸­å›½åšå£«åé¢ä¸Šé¡¹ç›®1é¡¹ï¼ŒçœåŸºç¡€ç ”ç©¶é¡¹ç›®2é¡¹ï¼Œâ€œå…´æ»‡è‹±æ‰æ”¯æŒè®¡åˆ’â€é’å¹´äººæ‰é¡¹ç›®1é¡¹ï¼Œäº‘å—çœæ™ºèƒ½ç³»ç»Ÿä¸è®¡ç®—é‡ç‚¹å®éªŒå®¤å¼€æ”¾è¯¾é¢˜ï¼ˆé‡ç‚¹é¡¹ç›®ï¼‰1é¡¹ç­‰ã€‚

<!--
ä¸‹é¢çš„å†…å®¹è¢«æ³¨é‡Šäº†ï¼š
è¿‘äº”å¹´ä¸»è¦ä»äº‹ç”Ÿç‰©ä¿¡æ¯è®¡ç®—ç ”ç©¶ï¼Œä¸»è¦è¯¾é¢˜ä¸ºâ€œé¢å‘ç™Œç—‡ç»„å­¦æ•°æ®èåˆçš„æ™ºèƒ½æ–¹æ³•ç ”ç©¶â€ï¼Œæå‡ºäº†å¤šç§ç»“æ„ç¨€ç–çŸ©é˜µå’Œæ·±åº¦å­¦ä¹ æ™ºèƒ½ç®—æ³•ï¼Œåº”ç”¨äºæ•´åˆåˆ†æå¤šç§ç™Œç—‡çš„å¤šå°ºåº¦ç»„å­¦æ•°æ®ï¼ŒæŒ–æ˜ç™Œç—‡å¼‚å¸¸ç”Ÿç‰©åˆ†å­åŠŸèƒ½æ¨¡å—ï¼Œå¯¹äºå‘ç°ç™Œç—‡é¶æ ‡å…·æœ‰é‡è¦æ„ä¹‰ã€‚å›´ç»•è¯¾é¢˜çš„å…³é”®ç§‘å­¦é—®é¢˜ï¼Œå·²å–å¾—è‰¯å¥½çš„ç§‘ç ”æˆæœï¼Œå‘è¡¨30å¤šç¯‡å­¦æœ¯è®ºæ–‡ï¼Œå…¶ä¸­ç¬¬ä¸€ä½œè€…æˆ–é€šè®¯ä½œè€…å‘è¡¨CCF A/Bç±»æœŸåˆŠ/ä¼šè®®20å¤šç¯‡ï¼ŒåŒ…æ‹¬IEEE TKDE (2021, 2023)ï¼ŒIEEE TNNLS 2022ï¼ŒBioinformatics (2018, 2023)ï¼ŒPLoS Computational Biology 2021ï¼ŒIEEE TCBB 2018ï¼ŒIEEE TNB 2016ç­‰é‡è¦æœŸåˆŠï¼Œä¸»æŒå›½å®¶åŸºé‡‘é¡¹ç›®2é¡¹ï¼Œä¸­å›½åšå£«åé¢ä¸Šé¡¹ç›®1é¡¹ï¼ŒçœåŸºç¡€ç ”ç©¶é¡¹ç›®2é¡¹ï¼Œâ€œå…´æ»‡è‹±æ‰æ”¯æŒè®¡åˆ’â€é’å¹´äººæ‰é¡¹ç›®1é¡¹ï¼Œäº‘å—çœæ™ºèƒ½ç³»ç»Ÿä¸è®¡ç®—é‡ç‚¹å®éªŒå®¤å¼€æ”¾è¯¾é¢˜ï¼ˆé‡ç‚¹é¡¹ç›®ï¼‰1é¡¹ç­‰ã€‚
-->

## äºŒã€ç ”ç©¶æ–¹å‘
<!--
&emsp;&emsp;<font color="blue"> é—µæ–‡æ–‡è€å¸ˆé•¿æœŸä»äº‹äººå·¥æ™ºèƒ½ç®—æ³•ï¼ˆåŸºäºTransformerçš„æ·±åº¦æ–¹æ³•ã€å›¾ç¥ç»ç½‘ç»œã€æ·±åº¦æ‰©æ•£æ¨¡å‹ã€å¯¹æ¯”å­¦ä¹ ã€å•ç»†èƒå¤§æ¨¡å‹ã€çŸ©é˜µå¼ é‡ä¼˜åŒ–ç­‰ï¼‰åŠå…¶åœ¨ç”Ÿç‰©åŒ»å­¦å¤§æ•°æ®åˆ†ææ–¹é¢çš„åº”ç”¨ç ”ç©¶ã€‚</font>
ä¸»è¦ç ”ç©¶æ–¹æ³•å¦‚ä¸‹ï¼š
-->

- æœºå™¨å­¦ä¹ ç†è®º  <br>
(1) æ·±åº¦å­¦ä¹ æ–¹æ³•ç ”ç©¶ï¼ˆå›¾ç¥ç»ç½‘ç»œã€æ‰©æ•£æ¨¡å‹ã€Transformerå˜ä½“ã€å¯¹æ¯”å­¦ä¹ ã€å•ç»†èƒå¤§æ¨¡å‹ç­‰ï¼‰<br>
(2) å¤šè§†å›¾æ— ç›‘ç£èšç±»æ–¹æ³•ç ”ç©¶ï¼ˆåŸºäºå›¾ç¥ç»ç½‘ç»œçš„æ–¹æ³•ç ”ç©¶ï¼‰ <br>
(3) çŸ©é˜µå¼ é‡ä¼˜åŒ–æ–¹æ³•ç ”ç©¶ï¼ˆéå‡¸ä¼˜åŒ–ã€L20åŠL0ç»“æ„ç¨€ç–ä¼˜åŒ–ã€ç‰¹å¾é€‰æ‹©ï¼‰<br>
- åŒ»å­¦äººå·¥æ™ºèƒ½ <br>
(1) AI + åŒ»å­¦å›¾åƒ <br>
(2) AI + å¤šæ¨¡æ€ç”Ÿç‰©åŒ»å­¦æ•°æ® <br>
 
<!--
(2) æ·±åº¦å­¦ä¹ åœ¨ç©ºé—´è½¬å½•ç»„å­¦åŠå•ç»†èƒç»„å­¦æ•°æ®èåˆä¸­çš„åº”ç”¨ç ”ç©¶
(3) æ·±åº¦å­¦ä¹ åœ¨åŒ»å­¦å›¾åƒå¤„ç†åŠå¤šæ¨¡æ€æ•°æ®èåˆä¸­çš„åº”ç”¨ç ”ç©¶<br>
(4) æ·±åº¦å­¦ä¹ åœ¨ç™Œç—‡ç»„å­¦æ•°æ®åˆ†æä¸­çš„åº”ç”¨ç ”ç©¶<br>
(5) æ·±åº¦å­¦ä¹ åœ¨å¾®ç”Ÿç‰©ç»„å­¦æ•°æ®åˆ†æä¸­çš„åº”ç”¨ç ”ç©¶<br>
-->

## ä¸‰ã€å­¦æœ¯è®ºæ–‡ 

- è°·æ­Œå­¦æœ¯ä¸»é¡µ: [Full List at My Google Scholar](https://scholar.google.com/citations?user=0Uy0GnoAAAAJ&hl=en) 
- ORCIDä¸»é¡µ: [My ORCID](https://orcid.org/0000-0002-2558-2911)
- ResearchGateä¸»é¡µ: [My ResearchGate](https://www.researchgate.net/profile/Wenwen-Min)

## å››ã€å­¦ç”ŸåŸ¹å…»

### æ‹›ç”Ÿå¹¿å‘Š
- æœ¬äººç›®å‰å¯ä»¥æ‹›æ”¶è®¡ç®—æœºç›¸å…³ä¸“ä¸šçš„ä¸“ç¡•ã€å­¦ç¡•
- æ‹›ç”Ÿè¦æ±‚
  - åšæŒä¸æ‡ˆçš„æ¯…åŠ›ï¼ˆæœ€çœ‹é‡çš„å“è´¨ï¼‰
  - è‰¯å¥½çš„ç¼–ç¨‹ã€æ•°å­¦ã€è‹±è¯­èƒ½åŠ›
  - ä¼˜å…ˆæ‹›æ”¶æƒ³è¯»åšçš„åŒå­¦ 
  - ç›®å‰æš‚æ—¶ä¸è€ƒè™‘æ‹›ç”Ÿå¤–ç±å­¦ç”Ÿ
- <font color="red"> æ„Ÿå…´è¶£çš„åŒå­¦å¯ä»¥ç»™æˆ‘é‚®ä»¶(wenwen.min@qq.com)ï¼›é‚®ä»¶å†…å®¹åŒ…æ‹¬å¦‚ä¸‹ä¿¡æ¯ï¼šä¸ªäººç®€å†+æœ¬ç§‘æˆç»©å•+è€ƒç ”æˆç»©+å¤è¯•ä¸Šæœºæˆç»©ã€‚ </font>
  
### å­¦ç”Ÿæƒ…å†µï¼ˆ2024/12/20ï¼‰
- 2022çº§ï¼Œæ*å®‡ï¼ˆ4ç¯‡ï¼Œç ”ç©¶ç”Ÿå›½å¥–2ä¸‡ï¼Œäº‘å¤§å²³è™¹å¥–å­¦é‡‘-èµ„åŠ©ç‡0.4%ï¼Œå‘è¡¨ä¸­ç§‘é™¢ä¸€åŒºBIB2025ï¼ŒCCF Bä¼šè®®ä¸‰ç¯‡ï¼Œä¿é€æ­¦æ±‰å¤§å­¦è¯»åšï¼‰
- 2022çº§ï¼Œç‰›*äº‘ï¼ˆ3ç¯‡ï¼Œå‘è¡¨ä¸­ç§‘é™¢äºŒåŒºä¸¤ç¯‡INSC2024, CSBJ2024ï¼ŒSpaBatchåœ¨æŠ•ï¼Œä¿é€æ¾³é—¨ç†å·¥å¤§å­¦è¯»åšï¼‰
- 2022çº§ï¼Œæ¨*å³°ï¼ˆ4ç¯‡ï¼Œå‘è¡¨CCF Cä¼šè®®ä¸€ç¯‡ISBRA2024ï¼Œä¸­ç§‘é™¢ä¸‰åŒºCJEä¸¤ç¯‡ï¼ŒTNNLSåœ¨æŠ•ï¼‰
- 2022çº§ï¼Œç‹*éœ‡ï¼ˆ2ç¯‡ï¼Œå‘è¡¨CCF Bä¼šè®®ä¸€ç¯‡ï¼Œå¦ä¸€ç¯‡è®ºæ–‡åœ¨æŠ•ï¼‰
- 2023çº§ï¼Œå²*å±‚ï¼ˆ5ç¯‡ï¼Œå‘è¡¨ä¸­ç§‘é™¢ä¸€åŒºä¸€ç¯‡BIBï¼ŒCCF Bä¼šè®®ä¸€ç¯‡ï¼ŒCCF Cä¼šè®®ä¸€ç¯‡ï¼ŒMethods2025ï¼ŒCCF Bç±»æœŸåˆŠåœ¨æŠ•Bioinformaticsï¼‰
- 2023çº§ï¼Œæ–¹*æµ·ï¼ˆ5ç¯‡ï¼Œçœå…é¡¹ç›®1.5ä¸‡ï¼Œå·²å‘å››ç¯‡ï¼šCCF Bä¼šè®®ä¸¤ç¯‡ï¼ŒCCF Cä¼šè®®ä¸€ç¯‡ï¼ŒCCF Bé¡¶åˆŠPLoS CBä¸€ç¯‡, ä¸€åŒºCB2025åœ¨æŠ•ï¼‰
- 2023çº§ï¼Œè–›*æ—ï¼ˆ5ç¯‡ï¼Œçœå…é¡¹ç›®1.5ä¸‡ï¼Œå‘è¡¨CCF Cä¼šè®®ä¸€ç¯‡ï¼Œä¸­ç§‘é™¢ä¸€åŒºä¸€ç¯‡BIB2025ï¼Œä¸­ç§‘é™¢ä¸€åŒºä¸€ç¯‡TST2025ï¼ŒMICCAI2025åœ¨æŠ•ï¼Œä¸€åŒºAS2025åˆæ­¥å®Œæˆï¼‰
- 2023çº§ï¼Œå²*æºï¼ˆ3ç¯‡ï¼Œå‘è¡¨CCF Bä¼šè®®ä¸€ç¯‡ï¼ŒJCBæœŸåˆŠä¸€ç¯‡ï¼Œä¸€åŒºBIBåœ¨æŠ•ï¼‰
- 2023çº§ï¼Œé»„*ç³ï¼ˆ2ç¯‡ï¼Œå‘è¡¨CCF Bä¼šè®®ä¸€ç¯‡ï¼ŒBIBåœ¨æŠ•ï¼‰
- 2023çº§ï¼Œèµµ*æˆï¼ˆ1ç¯‡ï¼ŒBIB2025åœ¨æŠ•ï¼‰
- 2024çº§ï¼Œèµµ*æ°ï¼ˆ1ç¯‡ï¼ŒIEEE JBHIåœ¨æŠ•ï¼‰
- 2024çº§ï¼Œé»„*ç…¦ï¼ˆè¿›è¡Œä¸­ï¼‰
- 2024çº§ï¼Œæœ*æ¢¦ï¼ˆè¿›è¡Œä¸­ï¼‰
- 2024çº§ï¼Œåˆ˜*ä¸¾ï¼ˆ1ç¯‡ï¼ŒIEEE JBHIåœ¨æŠ•ï¼‰
- 2024çº§ï¼Œæ*è¡¡ï¼ˆè¿›è¡Œä¸­ï¼‰
- 2024çº§ï¼Œç‹*æ¯…ï¼ˆè¿›è¡Œä¸­ï¼‰
- 2024çº§ï¼Œæ*å¥¥ï¼ˆè¿›è¡Œä¸­ï¼‰

## äº”ã€ä¸»è¦è®ºæ–‡
#åŒç­‰è´¡çŒ®ä½œè€…ï¼Œ*é€šè®¯ä½œè€…

### ğŸ”¥ 2025
- [38] __Wenwen Min*__, X Shi, et. al. SpaViT: Self-supervised Prediction of High-Resolution Spatial Transcriptomics with Vision Transformer. __<u> Tsinghua Science and Technology 2025 </u>__, in press

- [37] J Chen*, __Wenwen Min*__. sTPLS: identifying common and specific correlated patterns under multiple biological conditions. __<u> Briefings in Bioinformatics 2025 </u>__, DOI: 10.1093/bib/bbaf195, in press

- [36] W Yang, __Wenwen Min*__. Graph regularized sparse nonnegative Tucker decomposition with L0-constraints for unsupervised learning. __<u> Chinese Journal of Electronics 2025 </u>__, in press

- [35] W Yang, __Wenwen Min*__. Sparse nonnegative CP decomposition with graph regularization and L0-constraints for clustering. __<u> Chinese Journal of Electronics 2025 </u>__, in press

- [34] X Shi, F Zhu, __Wenwen Min*__. VTrans: A VAE-based Pre-trained Transformer Method for Microbiome Data Analysis. __<u> Journal of Computational Biology 2025 </u>__, in press

- [33] __Wenwen Min*__, D Fang, J Chen, S Zhang. SpaMask: Dual masking graph autoencoder with contrastive learning for spatial transcriptomics.  __<u> PLOS Computational Biology 2025 </u>__, DOI: 10.1371/journal.pcbi.1012881

- [32] S Xue, F Zhu, J Chen, __Wenwen Min*__. Inferring single-cell resolution spatial gene expression via fusing spot-based spatial transcriptomics, location and histology using GCN.  __<u> Briefings in Bioinformatics 2025 </u>__, DOI: 10.1093/bib/bbae630 (CCF Bç±»æœŸåˆŠ, ä¸­ç§‘é™¢JCR1åŒº, IF=9.5)

- [31] Z Shi, F Zhu, , __Wenwen Min*__. Inferring Multi-slice Spatially Resolved Gene Expression from H&E-stained Histology Images with STMCL.  __<u> Methods </u>__, 2025 (ä¸­ç§‘é™¢JCR3åŒº, IF=4.2)

### ğŸ”¥ 2024

- [30] __Wenwen Min*__, Zhiceng Shi, Jun Zhang, Jun Wan and Changmiao Wang. Multimodal contrastive learning for spatial gene expression prediction using histology images. __<u> Briefings in Bioinformatics 2024</u>__, DOI: 10.1093/bib/bbae551 (CCF Bç±»æœŸåˆŠ, ä¸­ç§‘é™¢JCR1åŒº, IF=9.5)

- [29] Xiaoyu Li, Fangfang Zhu and __Wenwen Min*__. SpaDiT: Diffusion Transformer for Spatial Gene Expression Prediction using scRNA-seq. __<u> Briefings in Bioinformatics 2024</u>__, DOI: 10.1093/bib/bbae571 (CCF Bç±»æœŸåˆŠ, ä¸­ç§‘é™¢JCR1åŒº, IF=9.5)
  
- [28] Jinyun Niu, Fangfang Zhu, TaoSheng Xu, Shunfang Wang and __Wenwen Min*__. Deep Clustering Representation for Spatially Resolved Transcriptomics Data via Multi-view Variational Graph Auto-Encoders with Consensus Clustering. __<u> Computational and Structural Biotechnology Journal 2024</u>__, https://doi.org/10.1016/j.csbj.2024.11.041 (ä¸­ç§‘é™¢JCR2åŒº, IF=4.4)

- [27] J Niu, F Zhu, D Fang, __Wenwen Min*__. SpatialCVGAE: Consensus Clustering Improves Spatial Domain Identification of Spatial Transcriptomics using VGAE. __<u> Interdisciplinary Sciences: Computational Life Sciences 2024  </u>__ (ä¸­ç§‘é™¢JCR2åŒº, IF=4.1)

- [26] X Shi, F Zhu, __Wenwen Min*__. Pretrained-Guided Conditional Diffusion Models for Microbiome Data Analysis. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%)

- [25] D Fang, F Zhu, __Wenwen Min*__. Multi-Slice Spatial Transcriptomics Data Integration Analysis with STG3Net. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%)

- [24] D Fang, F Zhu, D Xie, __Wenwen Min*__. Masked Graph Autoencoders with Contrastive Augmentation for Spatially Resolved Transcriptomics Data. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%)

- [23] __Wenwen Min*__, Z Wang, F Zhu, T Xu, S Wang scASDC: Attention Enhanced Structural Deep Clustering for Single-cell RNA-seq Data. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%)

- [22] Zhiceng Shi, S. L. Xue, F. Zhu and __Wenwen Min*__. High-Resolution Spatial Transcriptomics from Histology Images using HisToSGE with Pathology Image Large Model. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%)
 
- [21] Xiaoyu Li, __Wenwen Min*__, Shunfang Wang, Changmiao Wang and Taosheng Xu. stMCDI: Masked Conditional Diffusion Model with Graph Neural Network for Spatial Transcriptomics Data Imputation. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡20%)

- [20] L Huang, X Liu, S Wang, __Wenwen Min*__. Masked adversarial neural network for cell type deconvolution in spatial transcriptomics. __<u> BIBM 2024 </u>__ (CCF Bç±»ä¼šè®®)

- [19] Donghai Fang, Yichen Gao, Zhaoying Wang, Fangfang Zhu and __Wenwen Min*__. Contrastive Masked Graph Autoencoders for Spatial Transcriptomics Data Analysis. __<u> ISBRA 2024 </u>__ (CCF Cç±»ä¼šè®®)

- [18] Zhiceng Shi, Fangfang Zhu, Changmiao Wang and __Wenwen Min*__. Spatial Gene Expression Prediction from Histology Images with STco.__<u> ISBRA 2024 </u>__ (CCF Cç±»ä¼šè®®)

- [17] Shuailin Xue, Fangfang Zhu, Changmiao Wang and __Wenwen Min*__. stEnTrans: Transformer-based deep learning for spatial transcriptomics enhancement. __<u> ISBRA 2024 </u>__ (CCF Cç±»ä¼šè®®)
  
- [16] F. Yang, __Wenwen Min*__. Globally Convergent Accelerated Algorithms for Multilinear Sparse Logistic Regression with L0-Constraints. __<u>International Conference on Intelligent Computing (ICIC)}</u>__, 88-99, 2024 (CCF Cç±»ä¼šè®®)

- [15] Li Yu#, __Wenwen Min#__, Shunfang Wang\*. Boundary-Aware Gradient Operator Network for Medical Image Segmentation.
  __<u>IEEE Journal of Biomedical and Health Informatics</u>__,
  2024 (ä¸­ç§‘é™¢1åŒº, CCF Cç±»æœŸåˆŠ, IF=7.7)

### ğŸ”¥ 2023

- [14] Xiaoyu Li, Taosheng Xu, Jinyu Chen, Jun Wan and __Wenwen Min*__(å­¦ç”Ÿä¸€ä½œ). 
  Multimodal attention-based variational autoencoder for clinical risk prediction. 
  __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2023 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%)

- [13] Xiaoyu Li, __Wenwen Min*__(å­¦ç”Ÿä¸€ä½œ), Jinyu Chen, Jiaxin Wu and Shunfang Wang.
  TransVCOX: Bridging Transformer Encoder and Pre-trained VAE for Robust Cancer Multi-Omics Survival Analysis.
  __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2023 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%)

- [12] Weihua Zheng#, __Wenwen Min#__, Shunfang Wang\*. TsImpute: An accurate two-step imputation method for single-cell RNA-seq data.
  __<u>Bioinformatics</u>__,
  39(12):btad731, 2023 (CCF Bç±»æœŸåˆŠ, IF=5.8)

- [11] __Wenwen Min*__, Taosheng Xu, Xiang Wan, Tsung-Hui Chang.
  Structured Sparse Non-negative Matrix Factorization with L20-Norm.
  __<u>IEEE Transactions on Knowledge and Data Engineering</u>__,
  35(8):8584-8595, 2023
  (CCF Aç±»æœŸåˆŠ, IF=9.235) [[URL](https://ieeexplore.ieee.org/document/9893402),PDF,Data,Code] <br>

### ğŸ”¥ 2022

- [10] __Wenwen Min__, Xiang Wan, Tsung-Hui Chang, Shihua Zhang*.
  A Novel Sparse Graph-Regularized Singular Value Decomposition Model for Gene Co-Expression Pattern Discovery.
  __<u>IEEE Transactions on Neural Networks and Learning Systems</u>__,
  33(8):3842-3856, 2022
  (ä¸­ç§‘é™¢ä¸€åŒºï¼ŒCCF Bç±»æœŸåˆŠ, IF=14.255) [[URL](https://ieeexplore.ieee.org/document/9350193),PDF,Data,Code]<br>
  
- [09] __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Group-sparse SVD Models via L0 and L1-norm Penalties and Their Applications in Biological Data.
  __<u>IEEE Transactions on Knowledge and Data Engineering</u>__, 
  33(2):536-550, 2021 (CCF Aç±»æœŸåˆŠ, IF=9.235) [[URL](https://ieeexplore.ieee.org/document/8782829),PDF,Data,Code]<br>

- [08] __Wenwen Min__, Tsung-Hui Chang, Shihua Zhang\*, Xiang Wan*. 
  TSCCA: A tensor sparse CCA method for detecting microRNA-gene patterns from multiple cancers. 
  __<u>PLoS Computational Biology</u>__, 
  17(6):e1009044, 2021 
  (CCF Bç±»æœŸåˆŠ, IF=4.779)

- [07] F. Zhu*, J. Li*, J. Liu*, __Wenwen Min*__. Network-based cancer genomic data integration for pattern discovery. __<u>BMC Genomic Data</u>__, 22: Article 54, 2021 (SCIäºŒåŒº)
  
- [06] F. Zhu, J. Liu, __Wenwen Min*__. Gene Functional Module Discovery via Integrating Gene Expression and PPI Network Data. __<u>International Conference on Intelligent Computing (ICIC)}</u>__, 116-126, 2019 (CCF Cç±»ä¼šè®®)

- [05] __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Edge-group sparse PCA for network-guided high dimensional data analysis.
  __<u>Bioinformatics</u>__,
  34(20):3479-3487, 2018
  (CCF Bç±»æœŸåˆŠ, IF=6.931)

- [04] __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Network-regularized sparse logistic regression models for clinical risk prediction and biomarker discovery.
    __<u>IEEE/ACM Transactions on Computational Biology and Bioinformatics</u>__,
  15(3):944-953, 2018
  (CCF Bç±»æœŸåˆŠ, IF=4.5)

- [03] __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Sparse weighted canonical correlation Analysis.
  __<u>Chinese Journal of Electronics (ç”µå­å­¦æŠ¥è‹±æ–‡ç‰ˆ)</u>__,
  27(3):459-466, 2018 ï¼ˆCCF Top Iç±»æœŸåˆŠï¼‰

- [02] __Wenwen Min__, Juan Liu\*, Fei Luo, Shihua Zhang*.
  A two-stage method to identify joint modules from matched microRNA and mRNA expression data.
    __<u>IEEE Transactions on Nanobioscience</u>__,
  15(4):362-370, 2016 

- [01] __Wenwen Min__, Juan Liu*, Fei Luo, Shihua Zhang*.
  A novel two-stage method for identifying microRNA- gene regulatory modules in breast cancer.
    __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2015 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%ï¼‰

  

## å…­ã€ä¸“åˆ©å’Œè½¯è‘—
### ä¸“åˆ©
- é—µæ–‡æ–‡ï¼Œè–›å¸…æ—ç­‰. ä¸€ç§è¶…åˆ†è¾¨ç‡åŸºå› è¡¨è¾¾å›¾è°±çš„é¢„æµ‹æ–¹æ³•ã€è£…ç½®åŠè®¾å¤‡ï¼Œ[å®å®¡](https://patent.lotut.com/patent/66c6fdb65dbd827da424cc42.html)
- é—µæ–‡æ–‡ï¼Œå²é‘«æºç­‰. åŸºäºå¾®ç”Ÿç‰©ç»„æ•°æ®çš„ç”Ÿå­˜é£é™©é¢„æµ‹æ–¹æ³•ã€è£…ç½®ã€è®¾å¤‡åŠå­˜å‚¨ä»‹è´¨ï¼Œ[å®å®¡](https://patent.lotut.com/patent/66d040b15dbd825dd0adf28a.html)
- é—µæ–‡æ–‡ï¼Œå²å¿—å±‚ç­‰. åŸºäºç»„ç»‡å­¦å›¾åƒçš„ç©ºé—´åŸºå› è¡¨è¾¾æ°´å¹³é¢„æµ‹æ–¹æ³•ã€è£…ç½®åŠè®¾å¤‡ï¼Œ[å®å®¡](https://patent.lotut.com/patent/66e551a65dbd824d04f07d97.html)
- é—µæ–‡æ–‡ï¼Œç‹éœ‡. ä¸€ç§æ³¨æ„åŠ›å¢å¼ºæ·±åº¦å›¾åµŒå…¥çš„å•ç»†èƒRNA-Seqæ•°æ®èšç±»æ–¹æ³•ï¼Œå®å®¡
- é—µæ–‡æ–‡ï¼Œæ–¹ä¸œæµ·. ä¸€ç§åŸºäºå›¾ç¥ç»ç½‘ç»œçš„ç©ºé—´è½¬å½•ç»„å­¦å¤šåˆ‡ç‰‡ç©ºé—´åŸŸè¯†åˆ«åŠæ‰¹æ¬¡æ•ˆåº”å»é™¤æ–¹æ³•ï¼Œå®å®¡

### è½¯è‘—
- é—µæ–‡æ–‡ï¼Œæ–¹ä¸œæµ·ç­‰. ç©ºé—´è½¬å½•ç»„å­¦æ•°æ®èšç±»è½¯ä»¶V1.0
- é—µæ–‡æ–‡ï¼Œé—«ä¸€è¯ºï¼Œå²é‘«æºç­‰. ç™Œç—‡å¾®ç”Ÿç‰©ç»„æ•°æ®åˆ†æé¢„è®­ç»ƒTransformeræ¨¡å‹è½¯ä»¶V1.0
- é—µæ–‡æ–‡ï¼Œé—«ä¸€è¯ºï¼Œææ™“å®‡ç­‰. åŸºäºTransformerå’ŒVAEé¢„è®­ç»ƒçš„æ·±åº¦å­¦ä¹ ç”Ÿå­˜åˆ†ææ¨¡å‹è½¯ä»¶V1.0
- é—µæ–‡æ–‡ï¼Œå´ä½³æ¬£ç­‰. åŸºäºæ·±åº¦å­¦ä¹ çš„è„‘è‚¿ç˜¤æ™ºèƒ½è¾…åŠ©è¯Šç–—å¹³å° V1.0
- é—µæ–‡æ–‡ï¼Œå²å¿—å±‚ç­‰. åŸºäºæ·±åº¦å­¦ä¹ çš„è„‘è‚¿ç˜¤æ™ºèƒ½è¾…åŠ©è¯Šç–—å¹³å° V1.0
- é—µæ–‡æ–‡ï¼Œç‰›æ™‹äº‘ç­‰. ä¸€ç§åŸºäºå›¾æ·±åº¦å­¦ä¹ çš„ç©ºé—´è½¬å½•ç»„å­¦æ•°æ®èšç±»è½¯ä»¶ V1.0


<!-- ä¸‹é¢çš„å†…å®¹è¢«Markdown æ³¨é‡Š -->
<!--
## ä¸‰ã€è¿‘æœŸè®ºæ–‡ (2024)
You can also find my articles on [My Google Scholar Profile](https://scholar.google.com/citations?user=0Uy0GnoAAAAJ&hl=en) and <u>*Corresponding author</u> and <u>#Co-first author</u>

- __Wenwen Min*__, Donghai Fang, Jinyu Chen, and Shihua Zhang. "Dimensionality Reduction and Denoising of Spatial Transcriptomics Data Using Dual-Channel Masked Graph Autoencoder." bioRxiv (2024): 2024-05.


- Zhen Wang, and __Wenwen Min*__. "Graph Regularized NMF with L20-norm for Unsupervised Feature Learning." arXiv preprint arXiv:2403.10910 (2024).

- Weifeng Yang, and __Wenwen Min*__. "Graph regularized sparse nonnegative Tucker decomposition with â„“0-constraints for unsupervised learning."  submission to Applied Mathematical Modelling 2024

- Li Yu#, __Wenwen Min#__, Shunfang Wang\*. "Boundary-Aware Gradient Operator Network for Medical Image Segmentation." IEEE Journal of Biomedical and Health Informatics, 2024.


  
- Weifeng Yang,  __Wenwen Min*__. "Globally Convergent Accelerated Algorithms for Multilinear Sparse Logistic Regression with L0-Constraints." Lecture Notes in Computer Science (ICIC 2024), [URL](https://link.springer.com/chapter/10.1007/978-981-97-5663-6_8)

-->
  
<!-- ä¸‹é¢çš„å†…å®¹è¢«Markdown æ³¨é‡Š -->
<!--
## å››ã€ä»£è¡¨æ€§è®ºæ–‡
You can also find my articles on [My Google Scholar Profile](https://scholar.google.com/citations?user=0Uy0GnoAAAAJ&hl=en) and <u>*Corresponding author</u> and <u>#Co-first author</u>

- __Wenwen Min*__, Taosheng Xu, Xiang Wan, Tsung-Hui Chang.
  Structured Sparse Non-negative Matrix Factorization with L20-Norm.
  __<u>IEEE Transactions on Knowledge and Data Engineering</u>__,
  35(8):8584-8595, 2023
  (CCF Aç±»æœŸåˆŠ, IF=9.235) [[URL](https://ieeexplore.ieee.org/document/9893402),PDF,Data,Code] <br>

- __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Group-sparse SVD Models via L0 and L1-norm Penalties and Their Applications in Biological Data.
  __<u>IEEE Transactions on Knowledge and Data Engineering</u>__, 
  33(2):536-550, 2021
  (CCF Aç±»æœŸåˆŠ, IF=9.235) [[URL](https://ieeexplore.ieee.org/document/8782829),PDF,Data,Code]<br>

- __Wenwen Min__, Xiang Wan, Tsung-Hui Chang, Shihua Zhang*.
  A Novel Sparse Graph-Regularized Singular Value Decomposition Model for Gene Co-Expression Pattern Discovery.
  __<u>IEEE Transactions on Neural Networks and Learning Systems</u>__,
  33(8):3842-3856, 2022
  (ä¸­ç§‘é™¢ä¸€åŒºï¼ŒCCF Bç±»æœŸåˆŠ, IF=14.255) [[URL](https://ieeexplore.ieee.org/document/9350193),PDF,Data,Code]<br>
  
- __Wenwen Min__, Tsung-Hui Chang, Shihua Zhang\*, Xiang Wan*. 
  TSCCA: A tensor sparse CCA method for detecting microRNA-gene patterns from multiple cancers. 
  __<u>PLoS Computational Biology</u>__, 
  17(6):e1009044, 2021 
  (CCF Bç±»æœŸåˆŠ, IF=4.779)

- __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Edge-group sparse PCA for network-guided high dimensional data analysis.
  __<u>Bioinformatics</u>__,
  34(20):3479-3487, 2018
  (CCF Bç±»æœŸåˆŠ, IF=6.931)

- __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Network-regularized sparse logistic regression models for clinical risk prediction and biomarker discovery.
    __<u>IEEE/ACM Transactions on Computational Biology and Bioinformatics</u>__,
  15(3):944-953, 2018
  (CCF Bç±»æœŸåˆŠ, IF=4.5)

- __Wenwen Min__, Juan Liu\*, Fei Luo, Shihua Zhang*.
  A two-stage method to identify joint modules from matched microRNA and mRNA expression data.
    __<u>IEEE Transactions on Nanobioscience</u>__,
  15(4):362-370, 2016 

- __Wenwen Min__, Juan Liu*, Fei Luo, Shihua Zhang*.
  A novel two-stage method for identifying microRNA- gene regulatory modules in breast cancer.
    __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2015 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%ï¼‰

- __Wenwen Min__, Juan Liu\*, Shihua Zhang*.
  Sparse weighted canonical correlation Analysis.
  __<u>Chinese Journal of Electronics (ç”µå­å­¦æŠ¥è‹±æ–‡ç‰ˆ)</u>__,
  27(3):459-466, 2018 ï¼ˆCCF Top Iç±»æœŸåˆŠï¼‰

- Xiaoyu Li, Taosheng Xu, Jinyu Chen, Jun Wan and __Wenwen Min*__(å­¦ç”Ÿä¸€ä½œ, æœ¬äººé€šä¿¡). 
  Multimodal attention-based variational autoencoder for clinical risk prediction. 
  __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2023 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%)

- Xiaoyu Li, __Wenwen Min*__(å­¦ç”Ÿä¸€ä½œ, æœ¬äººé€šä¿¡), Jinyu Chen, Jiaxin Wu and Shunfang Wang.
  TransVCOX: Bridging Transformer Encoder and Pre-trained VAE for Robust Cancer Multi-Omics Survival Analysis.
  __<u>IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</u>__, 2023 (CCF Bç±»ä¼šè®®, æ¥æ”¶ç‡19%)

- Weihua Zheng#, __Wenwen Min#__, Shunfang Wang\*. TsImpute: An accurate two-step imputation method for single-cell RNA-seq data.
  __<u>Bioinformatics</u>__,
  39(12):btad731, 2023 (CCF Bç±»æœŸåˆŠ, IF=5.8)

- Li Yu#, __Wenwen Min#__, Shunfang Wang\*. Boundary-Aware Gradient Operator Network for Medical Image Segmentation.
  __<u>IEEE Journal of Biomedical and Health Informatics</u>__,
  2024 (ä¸­ç§‘é™¢1åŒº, CCF Cç±»æœŸåˆŠ, IF=7.7)
-->


<!-- ä¸‹é¢çš„å†…å®¹è¢«Markdown æ³¨é‡Š -->
<!--
## äº”ã€ä¸“åˆ©å’Œè½¯è‘—
### ä¸“åˆ©
- é—µæ–‡æ–‡ï¼Œè–›å¸…æ—ç­‰. ä¸€ç§è¶…åˆ†è¾¨ç‡åŸºå› è¡¨è¾¾å›¾è°±çš„é¢„æµ‹æ–¹æ³•ã€è£…ç½®åŠè®¾å¤‡
- é—µæ–‡æ–‡ï¼Œå²é‘«æºç­‰. åŸºäºå¾®ç”Ÿç‰©ç»„æ•°æ®çš„ç”Ÿå­˜é£é™©é¢„æµ‹æ–¹æ³•ã€è£…ç½®ã€è®¾å¤‡åŠå­˜å‚¨ä»‹è´¨
- é—µæ–‡æ–‡ï¼Œå²å¿—å±‚ç­‰. åŸºäºç»„ç»‡å­¦å›¾åƒçš„ç©ºé—´åŸºå› è¡¨è¾¾æ°´å¹³é¢„æµ‹æ–¹æ³•ã€è£…ç½®åŠè®¾å¤‡
- é—µæ–‡æ–‡ï¼Œç‹éœ‡. ä¸€ç§æ³¨æ„åŠ›å¢å¼ºæ·±åº¦å›¾åµŒå…¥çš„å•ç»†èƒRNA-Seqæ•°æ®èšç±»æ–¹æ³•
- é—µæ–‡æ–‡ï¼Œæ–¹ä¸œæµ·. ä¸€ç§åŸºäºå›¾ç¥ç»ç½‘ç»œçš„ç©ºé—´è½¬å½•ç»„å­¦å¤šåˆ‡ç‰‡ç©ºé—´åŸŸè¯†åˆ«åŠæ‰¹æ¬¡æ•ˆåº”å»é™¤æ–¹æ³•

### è½¯è‘—
- é—µæ–‡æ–‡ï¼Œæ–¹ä¸œæµ·ç­‰. ç©ºé—´è½¬å½•ç»„å­¦æ•°æ®èšç±»è½¯ä»¶V1.0
- é—µæ–‡æ–‡ï¼Œé—«ä¸€è¯ºï¼Œå²é‘«æºç­‰. ç™Œç—‡å¾®ç”Ÿç‰©ç»„æ•°æ®åˆ†æé¢„è®­ç»ƒTransformeræ¨¡å‹è½¯ä»¶V1.0
- é—µæ–‡æ–‡ï¼Œé—«ä¸€è¯ºï¼Œææ™“å®‡ç­‰. åŸºäºTransformerå’ŒVAEé¢„è®­ç»ƒçš„æ·±åº¦å­¦ä¹ ç”Ÿå­˜åˆ†ææ¨¡å‹è½¯ä»¶V1.0
- é—µæ–‡æ–‡ï¼Œå´ä½³æ¬£ç­‰. åŸºäºæ·±åº¦å­¦ä¹ çš„è„‘è‚¿ç˜¤æ™ºèƒ½è¾…åŠ©è¯Šç–—å¹³å° V1.0
- é—µæ–‡æ–‡ï¼Œå²å¿—å±‚ç­‰. åŸºäºæ·±åº¦å­¦ä¹ çš„è„‘è‚¿ç˜¤æ™ºèƒ½è¾…åŠ©è¯Šç–—å¹³å° V1.0
- é—µæ–‡æ–‡ï¼Œç‰›æ™‹äº‘ç­‰. ä¸€ç§åŸºäºå›¾æ·±åº¦å­¦ä¹ çš„ç©ºé—´è½¬å½•ç»„å­¦æ•°æ®èšç±»è½¯ä»¶ V1.0
-->


<!-- ä¸‹é¢çš„å†…å®¹è¢«Markdown æ³¨é‡Š -->
<!--
## News
- 2024/10/12: 8 papers are accepted in BIBM 2024 (CCF B, acceptance rate is about 20%)
- 2024/05/15: the paper "Boundary-aware Gradient Operator Network for Medical Image Segmentation" is accepted in IEEE Journal of Biomedical and Health Informatics
- 2024/05/10: 5 papers are accepted in ISBRA 2024 (CCF C, acceptance rate is about 38%)
  - "stEnTrans: Transformer-based deep learning for spatial transcriptomics enhancement" with è–›å¸…æ—ï¼ˆä¸“ç¡•2023çº§ï¼‰
  - "Contrastive Masked Graph Autoencoders for Spatial Transcriptomics Data Analysis" with æ–¹ä¸œæµ·ï¼ˆå­¦ç¡•2023çº§ï¼‰
  - "Spatial gene expression prediction from histology images with STco" with å²å¿—å±‚ï¼ˆä¸“ç¡•2023çº§ï¼‰
  - "VTrans: A VAE-based Pre-trained Transformer Method for Microbiome Data Analysis" with å²é‘«æºï¼ˆä¸“ç¡•2023çº§ï¼‰
  - "SpatialCVGAE: Spatial Domain Identification via Consensus Clustering Integrated Variational Graph Autoencoder" with ç‰›æ™‹äº‘ï¼ˆä¸“ç¡•2022çº§ï¼‰
- 2024/05/09: 1 papers is accepted in ICIC 2023 (CCF B, acceptance rate is about NA)
  - "Globally Convergent Accelerated Algorithms for Multilinear Sparse Logistic Regression with L0-constraints" with æ¨ä¼Ÿå³°ï¼ˆå­¦ç¡•2022çº§ï¼‰
- 2023/12/16: the paper "TsImpute: An accurate two-step imputation method for single-cell RNA-seq data" is accepted in Bioinformatics (CCF B)
- 2023/10/14: 2 papers are accepted in BIBM 2023 (CCF B, acceptance rate is about 19%)
  - "TransVCOX: Bridging Transformer Encoder and Pre-trained VAE for Robust Cancer Multi-Omics Survival Analysis" with ææ™“å®‡ï¼ˆå­¦ç¡•2022çº§ï¼‰
  - "Multimodal attention-based variational autoencoder for clinical risk prediction" with ææ™“å®‡ï¼ˆå­¦ç¡•2022çº§ï¼‰
- 2022/09/10: the paper "Structured Sparse Non-negative Matrix Factorization with L20-Norm" is accepted as a regular paper in IEEE TKDE (CCF A)
- 2021/12/25: Joined Yunnan University.
-->
